jsk_interactive_markers/jsk_interactive_marker/src/marker_6dof.cpp

```diff
@@ -255,7 +278,9 @@ class Marker6DOF {
     pose.header = feedback->header;
     pose.pose = feedback->pose;
     latest_pose_ = pose;
-    pose_pub_.publish(pose);
+    if (!publish_pose_periodically_) {
```

Comment:
publish function at https://github.com/jsk-ros-pkg/jsk_visualization/pull/657/files/13b5a38ddf4c47c30930e387da2c1632b12b49ab#diff-51e3d57b1c1b9f815cd7be461b98acb4R306 is called periodically, so it is enabled only when publish_pose_periodically_ is true.